toplevel/vernac.ml

```diff
@@ -247,105 +242,3 @@ and load_vernac ~verbosely ~check ~interactive doc sid file =
 let process_expr doc sid loc_ast =
   checknav_deep loc_ast;
   interp_vernac ~interactive:true ~check:true doc sid loc_ast
-
-let warn_file_no_extension =
```

Comment:
Did this warning go away?

Reply:
No, moved to `coqtop`.

Reply:
This patch should be "semantics" preserving except for the bug fixed. So even non-optimal but non-buggy things are preserved indeed.

Reply:
Sorry, I was tricked by the diff being cut. Thanks!

Reply:
I think it is a very good question, in fact, after this `toplevel/vernac` is almost empty [most of the code there is just workarounds over STM limitations in order to imitate more legacy behavior] and should go away at some point with `coqloop` calling the stm directly in a small loop.

Now, `toplevel/coqtop` has inherited some of the "compile" file part, but this code should also gradually be taken over by other components as they assimilate the save/load document functionality.